### PR TITLE
DRAFT Feat/ghpc venn diagram

### PR DESCRIPTION
**Pull request by KaiElwood**
Draft PR request. Do not merge!

### CODE REVIEW

**Review by jdromero88 — CHANGES_REQUESTED** [review]
Hey @KaiElwood. Great job on this parce!! I left you 2 comments.

When you submit the final PR don't forget to remove all the `console.log()`s & to convert all the margins and paddings to `rem`s.

Tip: you can link the PR to an issue in different ways. By doing this when the PR is approved and merged it will close the issue automatically. You can see how to link them [here](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)

**Comment by KaiElwood** [comment]
Final edits have been made, ready for final review

File: ghpc-TB-venn-diagram/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+.highcharts-point {
+    stroke: none;
+}
+
+.highcharts-text-outline {
+    stroke-width: 0;
+    fill: white;
+    color: white;
+    visibility: visible;
+    text-shadow: 0px 0px 10px #00000040;
```
Comment (jdromero88) [inline ghpc-TB-venn-diagram/css/styles.css:26]: For consistency we should remove the px on 0 values.
```suggestion
    text-shadow: 0 0 10px #00000040;
```

**Review by jdromero88 — CHANGES_REQUESTED** [review]
Great job @KaiElwood!!! the diagram looks AMAZING!!
I left you one comment after that let me know and I will merge it.

**Pull request merged**